lib/chef/provisioning/convergence_strategy/install_msi.rb

```diff
@@ -9,8 +10,6 @@ def initialize(convergence_options, config)
         super
         @chef_version ||= convergence_options[:chef_version]
         @prerelease ||= convergence_options[:prerelease]
-        @install_msi_url = convergence_options[:install_msi_url] || 'https://www.chef.io/chef/install.msi'
```

Comment:
If we just stop using these options, we will have to move this change to a major release version.  Otherwise we need to support them for backwards compatibility.  Does `Mixlib::Install` not have analogous configs?